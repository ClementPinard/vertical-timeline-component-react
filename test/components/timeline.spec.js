import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

import Timeline from '../../src/components/timeline';

Enzyme.configure({ adapter: new Adapter() });

describe('<Timeline />', () => {
	it('should create Snapshot', () => {
		const component = renderer.create(<Timeline>childen</Timeline>);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});

	it('should render component', () => {
		const wrapper = shallow(<Timeline>Hello</Timeline>);
		expect(wrapper.children).toBeInstanceOf(Function);
	});
});
