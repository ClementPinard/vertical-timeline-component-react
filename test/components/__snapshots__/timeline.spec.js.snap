// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Timeline /> should create Snapshot 1`] = `
<div
  className="sc-AxjAm bXdGBi"
>
  <div
    className="sc-AxirZ lhnuda"
  >
    <div>
      Child
    </div>
  </div>
</div>
`;
